dicom-archive/DICOM/DCMSUM.pm

```diff
@@ -133,8 +133,7 @@ QUERY
 
     # INSERT or UPDATE 
     # get acquisition metadata
-    my $sfile = "$self->{tmpdir}/$meta.meta";
-    my $metacontent = &read_file($sfile);
+    my $metacontent = &read_file($metafile);
```

Comment:
Before your PR, the file that was read was `$self->{tmpdir}/$meta.meta` and now it is `$metafile`. The code is written as if it was right to assume that `$metafile` is already in `$self->{tmpdir}` and has extension `.meta`. Are you sure this is right? It probably is and I am most likely missing something since I am not super familiar with the code...

Reply:
Yes, I changed the value $meta to be $metafile that is already in the tmp directory.

Reply:
I also changed the value passed to the function in dicomTar.pl so that it is indeed the metafile in tmpdir. Hope my answer makes sense.

Reply:
I looked at `dicomTar.pl` and `$metafile` is set to `$targetLocation/$tarbasename.meta`. But `$targetLocation` seems to be a command line argument, so there is no guarantee that it will be set to tmpdir. Am I missing something (most likely yes...)?